README.md

```diff
@@ -16,7 +16,7 @@ git clone https://github.com/stfc/terraform-openstack.git
 
 ### Set up Access to the STFC cloud
  
-Copy your `PROJECT.rc` file onto the VM, then run `source PROJECT.rc` and enter your fedID password.
+Copy your `clouds.yaml` file into the VM's `.config/openstack/`. You will need to hard-code your password under `username:`. Some additional documentation can be found here: https://stfc-cloud-docs.readthedocs.io/en/latest/Reference/PythonSDK.html#clouds-yaml 
```

Comment:
Sorry, I'm going to be a PITA
We should be steering people to app credentials for this instead. We have brand new docs here for this we should point people to instead:
https://stfc-cloud-docs.readthedocs.io/en/latest/Keystone/ApplicationCredentials.html